src/sentry/static/sentry/app/views/organizationGroupDetails/groupSimilarIssues/similarTraceID/list.tsx

```diff
@@ -0,0 +1,204 @@
+import React from 'react';
+import {Location, Query} from 'history';
+import {browserHistory} from 'react-router';
+import pick from 'lodash/pick';
+import * as Sentry from '@sentry/react';
+import styled from '@emotion/styled';
+
+import {tct} from 'app/locale';
+import {URL_PARAM} from 'app/constants/globalSelectionHeader';
+import {stringifyQueryObject, QueryResults} from 'app/utils/tokenizeSearch';
+import GroupListHeader from 'app/components/issues/groupListHeader';
+import {Panel, PanelBody} from 'app/components/panels';
+import StreamGroup from 'app/components/stream/group';
+import GroupStore from 'app/stores/groupStore';
+import Pagination from 'app/components/pagination';
+import withApi from 'app/utils/withApi';
+import LoadingIndicator from 'app/components/loadingIndicator';
+import {Client} from 'app/api';
+import {Group} from 'app/types';
+import EmptyStateWarning from 'app/components/emptyStateWarning';
+import LoadingError from 'app/components/loadingError';
+import DateTime from 'app/components/dateTime';
+
+export type Issue = {
+  id: string;
+  trace: string;
+  ['issue.id']: number;
+};
+
+type CustomGroup = Group & {
+  eventID: string;
+  groupID: string;
+};
+
+type Period = {
+  start: string;
+  end: string;
+};
+
+type Props = {
+  api: Client;
+  orgSlug: string;
+  issues: Array<Issue>;
+  period: Period;
+  pageLinks: string | null;
+  location: Location;
+  traceID: string;
+  eventID: string;
+};
+
+type State = {
+  groups: Array<CustomGroup>;
+  hasError: boolean;
+  isLoading: boolean;
+};
+
+class List extends React.Component<Props, State> {
+  state: State = {
+    groups: [],
+    hasError: false,
+    isLoading: true,
+  };
+
+  componentDidMount() {
+    this.getGroups();
+  }
+
+  getGroups = async () => {
+    const {api, orgSlug, location, traceID, eventID, issues} = this.props;
+
+    if (!issues.length) {
+      this.setState({isLoading: false});
+      return;
+    }
+
+    const issuesIds = issues.map(issue => `group=${issue['issue.id']}`).join('&');
```

Comment:
I am passing this info to the URL as the endpoint only accepts numbers...We should update it on the backend.